help/destinations/catalog/advertising/magnite-batch.md

```diff
@@ -1,25 +1,29 @@
 ---
-title: Magnite Batch Destination
-description: Use this destination to deliver Adobe CDP audiences to the Magnite Streaming platform in batch.
+title: Magnite Batch destination
+description: Use this destination to deliver Adobe CDP audiences to the Magnite platform in batch.
 last-substantial-update: 2024-11-18
 exl-id: 8cc3890f-84f8-49d1-a329-322c13f9e5af
 ---
-# Magnite: Batch connection {#magnite-streaming-batch}
+# Magnite: Batch destination {#magnite-batch}
 
 ## Overview {#overview}
 
-This document describes the Magnite: Batch destination and provides sample use cases to help you better understand how to activate and export audiences to it.
+This document describes the [!DNL Magnite: Batch] destination and provides sample use cases to help you better understand how to activate and export audiences to it.
 
-Adobe Real-Time CDP audiences can be delivered to the Magnite Streaming platform in two ways - they can be delivered once per day, or they can be delivered in real-time:
+>[!NOTE]
+>
+>The [!DNL Magnite] destinations can be used for ingestion into any [!DNL Magnite] platform: Streaming, DV+, SpringServe, etc.
 
-1. If you only want and/or need to deliver audiences once per day, you can use the Magnite: Batch destination, which delivers audiences to Magnite Streaming via a daily S3 batch file delivery. These Batch audiences are stored indefinitely in the Magnite platform, unlike real-time audiences, which are only stored for a couple days.
+Adobe CDP audiences can be delivered to the [!DNL Magnite] platform in two ways - they can be delivered once per day, or they can be delivered in real-time:
```

Comment:
Please update all "Adobe CDP" occurences to "Adobe Real-Time CDP". Same for "CDP".